cdc/model/changefeed.go

```diff
@@ -33,8 +32,12 @@ type ChangeFeedDetail struct {
 	// Start sync at this commit ts if `StartTs` is specify or using the CreateTime of changefeed.
 	StartTs uint64 `json:"start-ts"`
 	// The ChangeFeed will exits until sync to timestamp TargetTs
-	TargetTs uint64          `json:"target-ts"`
-	Info     *ChangeFeedInfo `json:"-"`
+	TargetTs uint64 `json:"target-ts"`
+	// ChangeFeed can be resumed from stop state, this field is used to trigger a watch event in each capture
+	ResumeTs uint64 `json:"resume-ts"`
```

Comment:
Is it possible to use `checkpointTs` to avoid introducing a similar concept?

Reply:
removed, PTAL